src/pages/Home.js

```diff
@@ -24,21 +27,25 @@ const TopDecal = styled.img`
 `
 
 export default withTheme(({ announcements, theme }) => {
-  // const { user, isAuthed } = useAuth()
+  const { user, isAuthed } = useAuth()
 
   return (
     <HomeContainer>
       {theme.name === 'cmdf' && <TopDecal src={head_decal} />}
       {/* {theme.name === 'hackCamp' && <Hackcamp2023BG />} */}
       <HackerCountdown />
-      <Livestream />
       {/* 
       <CommonLinks />
       <Announcements announcements={announcements} /> */}
 
-      {/* {user?.status === APPLICATION_STATUS.accepted && isAuthed && user.uid && (
+      {/* Only display QR Code if logged in */}
+      {user?.status === APPLICATION_STATUS.accepted && isAuthed && user.uid ? (
         <QrCode userInfo={user} userId={user.uid} />
-      )} */}
+      ) : (
+        <p style={{ textAlign: 'center' }}>
+          Please login with the account accepted into nwHacks 2024
+        </p>
```

Comment:
Might want to use a component from Typography here and change the blurb to say "Please login with the email you used to apply to nwHacks 2024."